frontend/src/App.js

```diff
@@ -77,7 +78,25 @@ function App() {
     console.info('You clicked the Chip.');
     // getStockData(x);
   };
-  
+
+  // Mock Stock Card Data
+  const mockStockCardData = {
+    AAPL: {
+      name: "Apple Inc",
+      ticker: "AAPL",
+      price: "168.22",
+      priceChange: "0.54",
+      percentChange: "+1.19",
+    },
+    AMZN: {
+      name: "Amazon.com, Inc",
+      ticker: "AMZN",
+      price: "127.74",
+      priceChange: "1.33",
+      percentChange: "-0.82",
+    },
+  };
```

Comment:
This is only temporary. I can't seem to hit the yFinance API, results in a 404 but essentially when we have raw data to use, we can just use it to populate the fields.

Ideally imo, having TypeScript in the project would be nice as I've found that it helps with data modelling and enforcing it.